schema.xml

```diff
@@ -0,0 +1,1184 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
Did you intend to commit this?  I'm pretty sure this is generated on solr build and we shouldn't put it in source control.

Reply:
We can't commit it until the development environment supports SOLR schema update. I have converted it to a draft.

Reply:
I removed schema.xml.